src/format_url.js

```diff
@@ -12,6 +12,8 @@ module.exports = function(path, accessToken) {
     }
 
     var url = (document.location.protocol === 'https:' || config.FORCE_HTTPS) ? config.HTTPS_URL : config.HTTP_URL;
+    var vDupe = url.match('\/v[0-9]');
```

Comment:
Specifically, I'd replace these two lines with:

```
url = url.replace(/\/v4$/, '');
```

Only match `/v4`, only at the end of the string.